hpcflow/archive/cloud/dropbox_cp.py

```diff
@@ -0,0 +1,369 @@
+import os
```

Comment:
Could we add this module (and the later `onedrive` module) back into a `providers` sub-package? The class hierachy is then clear without opening the files.